packages/tests/src/helpers/generateMassiveProject.ts

```diff
@@ -71,8 +72,8 @@ export const generateProject = async (
     const target = path.join(__dirname, `../${PROJECT_PATH}`);
 
     // Clean previous project.
-    await fs.remove(target);
-    await fs.mkdir(target);
+    await rm(target);
+    await fs.mkdir(target, { recursive: true });
```

Comment:
Nit: shouldn't you make a helper for this too?

Reply:
That's fair.

Reply:
(no need to be done in this PR)